Userland/Libraries/LibJS/Runtime/Temporal/CalendarPrototype.cpp

```diff
@@ -648,4 +655,43 @@ JS_DEFINE_NATIVE_FUNCTION(CalendarPrototype::era_year)
     VERIFY_NOT_REACHED();
 }
 
+i32 CalendarPrototype::iso_year(Object& temporal_object)
```

Comment:
Just make these `static` standalone functions

Reply:
Done, I did that first, but I hated a little their new place at the start of the file instead of starting the spec comment.